test_conformance/c11_atomics/test_atomics.cpp

```diff
@@ -1202,7 +1202,8 @@ class CBasicTestFetchAdd
           min_range(-999.0), max_range(999.0), max_error(0.0)
     {
         if constexpr (
-            std::is_same_v<
+            std::is_same<HostDataType, HOST_HALF>::value
```

Comment:
Nitpick: I'd prefer to use `std::is_same_v` consistently.

```suggestion
            std::is_same_v<HostDataType, HOST_HALF>
```